wildfly-modules/jboss/container/wildfly/galleon-wildfly/artifacts/opt/jboss/container/wildfly/galleon/wildfly-s2i-galleon-pack/pom.xml

```diff
@@ -33,9 +33,10 @@
             <distribution>repo</distribution>
         </license>
     </licenses>
-    
+
     <properties>
         <version.org.wildfly>${env.WILDFLY_VERSION}</version.org.wildfly>
+        <version.org.wildfly.cloud.legacy.galleon-pack>${env.WILDFLY_LEGACY_OS_GALLEON_PACK_VERSION}</version.org.wildfly.cloud.legacy.galleon-pack>
```

Comment:
could you please rename the env to `WILDFLY-CLOUD_LEGACY_GALLEON_PACK_VERSION` to be consistent with the GAV?